src/apps/bridge/aanderaaController.cpp

```diff
@@ -152,15 +154,21 @@ void aanderaControllerInit(BridgePowerController *power_controller,
   configASSERT(sys_cfg);
   _ctx._bridge_power_controller = power_controller;
   _ctx._usr_cfg = usr_cfg;
+  _ctx._sys_cfg = sys_cfg;
 
   uint32_t agg_period_ms = (DEFAULT_CURRENT_AGG_PERIOD_MIN * 60 * 1000);
   uint32_t agg_period_min;
   if (_ctx._usr_cfg->getConfig("currentAggPeriodMin", strlen("currentAggPeriodMin"),
                                agg_period_min)) {
     agg_period_ms = (agg_period_min * 60 * 1000);
   }
+
+  _ctx.current_reading_period_ms = DEFAULT_CURRENT_READING_PERIOD_MS;
+  _ctx._sys_cfg->getConfig("cuurentReadingPeriodMs", strlen("currentReadinPeriodMs"), _ctx.current_reading_period_ms);
```

Comment:
Let's make this a constexpr or define... to prevent exactly the bug you've got here with a typo. 😜